backend/FwLite/FwLiteWeb/appsettings.sdk.json

```diff
@@ -6,7 +6,8 @@
     "OpenBrowser": false
   },
   "FwDataBridge": {
-    "ProjectsFolder": "./fw-projects"
+    "ProjectsFolder": "./fw-projects",
+    "TemplatesFolder": "."
```

Comment:
This would have been my gut choice.
```suggestion
    "TemplatesFolder": "./Templates"
```

Reply:
I thought of that, but it's always just going to be empty, so there's not much point